metaci/build/templates/build/detail_layout.html

```diff
@@ -1,4 +1,26 @@
 {% extends 'layout_full.html' %}
+{% load static %}
+
+{% block base_title %}
+    MetaCI - Build #{{ build.id }}
+{% endblock %}
+
+{% block base_head %}
+    {{ block.super }}
+    <meta name="twitter:card" content="summary" />
+    <meta name="og:type" content="article" />
+    <meta property="og:title" content="MetaCI - Build #{{ build.id }}" />
```

Comment:
Should we include the repo name in the title?

Reply:
Not sure. Seems like most platforms will truncate `title` to 40-55 characters. It should probably be in the summary at least. I'll add it there.